### PR TITLE
getdns 1.1.1, remove devel, use head do block

### PR DESCRIPTION
**Pull request by joelpurra**
## getdns 1.1.1

See [getdns v1.1.1 release notes](https://getdnsapi.net/releases/getdns-1-1-1/).

## getdns remove devel for now

There currently is no release candidate.

## getdns use head do with url

As [mentioned in the formula cookbook](http://docs.brew.sh/Formula-Cookbook.html), blocks are preferred over conditionals.

### CODE REVIEW

**Comment by joelpurra** [comment]
Fixed rubocop errors in https://github.com/Homebrew/homebrew-core/pull/14656/commits/d51ea1c5de4884549b03cba6c280ca5216999711

```text
Error: 3 problems in 1 formula
getdns:
  * libtool dependency should be
      depends_on "libtool" => :build
    Or if it is indeed a runtime dependency
      depends_on "libtool" => :run
  * autoconf dependency should be
      depends_on "autoconf" => :build
    Or if it is indeed a runtime dependency
      depends_on "autoconf" => :run
  * automake dependency should be
      depends_on "automake" => :build
    Or if it is indeed a runtime dependency
      depends_on "automake" => :run
```

**Comment by joelpurra** [comment]
Fixed style check offense in https://github.com/Homebrew/homebrew-core/pull/14656/commits/96090946e3878dbb4ac3a74a0070b5c7c0fdeaa4

```text
        == /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/getdns.rb ==
C: 20:  3: head (line 20) should be put before depends_on (line 13)

1 file inspected, 1 offense detected
```

File: Formula/getdns.rb [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
     sha256 "f9259f55b25aa8c20a880c90c63fbbbc2a33da2b68a617799d43d0ae7abcacb1" => :yosemite
   end
 
-  devel do
-    url "https://getdnsapi.net/releases/getdns-1-1-1rc1/getdns-1.1.1rc1.tar.gz"
-    sha256 "f63340b1d05410b875217c6abd7066586fc55a811db4ae90ffd01d2240e05e57"
+  head do
+    url "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+    depends_on "libtool" => :build
```
Comment (ilovezfs) [inline Formula/getdns.rb]: please move this below automake

Comment (joelpurra) [inline Formula/getdns.rb]: @ilovezfs: moved in d682fe348. Was that how you meant it?

Comment (joelpurra) [inline Formula/getdns.rb]: @ilovezfs: and if the move was correct, was it to get the list alphabetized? =)

Comment (ilovezfs) [inline Formula/getdns.rb]: Yep! Also, general consistency because they are almost always together and in this order.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks @joelpurra! Shipped :rocket:

**Pull request closed without merging**